main/firmware-linux/template.py

```diff
@@ -107,6 +109,13 @@
     ),
     ("intel-bt", "Intel Bluetooth", _arch_x86, "network", ["intel/ibt*"]),
     ("intel-ice", "Intel E800 series", _arch_x86, "network", ["intel/ice"]),
+    (
+        "intel-ivsc",
+        "Intel Visual Sensing Controller",
+        _arch_x86,
```

Comment:
this is new *lake only afaict so this should be accurate https://github.com/intel/ivsc-driver

also for reference https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/?id=cfc997c664f076c6800b749ed7ef315b1e8e76fd